src/app.py

```diff
@@ -224,7 +249,8 @@ async def populate():
             state.scene.add_entity(
                 Object(id=id, name= entity_name, position=Vector(x, y)))
             id += 1
-
+    task = asyncio.create_task(create_tick())
```

Comment:
It should be done with the first `get` instead IMHO

Reply:
Imo after populate, because we `get` everytime even when we don't have map.